tests/zfs-tests/tests/functional/online_offline/online_offline_002_neg.ksh

```diff
@@ -30,6 +30,7 @@
 #
 
 . $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/online_offline/online_offline.cfg
```

Comment:
I don't follow.  Why should it matter for a physical disk config if the default values appear in `online_offline.cfg` or the test case itself?